In [1]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [2]:
import pandas as pd
import requests
from bs4 import BeautifulSoup

In [3]:
df = pd.read_csv('/content/drive/MyDrive/data/busan/지역별_관광지_검색순위.csv')
df2 = pd.read_csv('/content/drive/MyDrive/data/busan/지역별_관광지_검색순위_식음료.csv')
df3 = pd.read_csv('/content/drive/MyDrive/data/busan/지역별_관광지_검색순위_숙박.csv')
df4 = pd.read_csv('/content/drive/MyDrive/data/busan/부산광역시_지역화폐(동백전) 가맹점 현황_20221222.csv', encoding = 'cp949')

In [4]:
df['중분류 카테고리'].value_counts()

문화관광     183
쇼핑       146
자연관광      62
기타관광      40
레저스포츠     31
역사관광      29
체험관광       9
Name: 중분류 카테고리, dtype: int64

# 중분류 카테고리 별 데이터 분류

In [10]:
df_ct = df[df['중분류 카테고리'] == '문화관광']
df_ct

순위        관광지명  광역시/도 시/군/구                도로명주소 중분류 카테고리 소분류 카테고리  \
13    14  롯데월드어드벤처부산  부산광역시   기장군   부산 기장군 동부산관광로 42-0     문화관광     테마공원   
27    28       BEXCO  부산광역시  해운대구   부산 해운대구 APEC로 55-0     문화관광     전시시설   
30    31    벡스코제1전시장  부산광역시  해운대구   부산 해운대구 APEC로 55-0     문화관광     전시시설   
35    36      부산시민공원  부산광역시  부산진구   부산 부산진구 시민공원로 73-0     문화관광     도시공원   
46    47       사직야구장  부산광역시   동래구      부산 동래구 사직로 45-0     문화관광  레저스포츠시설   
..   ...         ...    ...   ...                  ...      ...      ...   
491  492        75광장  부산광역시   영도구               부산 영도구     문화관광     도시공원   
494  495      조은극장1관  부산광역시    중구    부산 중구 구덕로34번길 4-0     문화관광     공연시설   
495  496     메가박스해운대  부산광역시  해운대구   부산 해운대구 해운대로 813-0     문화관광     공연시설   
497  498     메가박스부산대  부산광역시   금정구  부산 금정구 장전로12번길 55-0     문화관광     공연시설   
499  500      비콘그라운드  부산광역시   수영구     부산 수영구 망미번영로 0-0     문화관광  기타문화관광지   

        검색건수  
13   134,067  
27    89,716  
30    87,725  
35    78,895  
46    70,133  
..       ...  
491    2,060  
494    2,040  
495    2,033  
497    2,029  
499    2,015  

[183 rows x 8 columns]

In [11]:
df_shop = df[df['중분류 카테고리'] == '쇼핑']
df_shop

순위           관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리  \
0      1    신세계백화점센텀시티점  부산광역시  해운대구      부산 해운대구 센텀남대로 35-0       쇼핑   
2      3      롯데백화점부산본점  부산광역시  부산진구      부산 부산진구 가야대로 772-0       쇼핑   
6      7     코스트코홀세일부산점  부산광역시   수영구        부산 수영구 구락로 137-0       쇼핑   
9     10       롯데백화점광복점  부산광역시    중구          부산 중구 중앙대로 2-0       쇼핑   
10    11  롯데프리미엄아울렛동부산점  부산광역시   기장군      부산 기장군 기장해안로 147-0       쇼핑   
..   ...            ...    ...   ...                     ...      ...   
470  471     다이소부산화명2호점  부산광역시    북구        부산 북구 금곡대로 208-0       쇼핑   
475  476      반여3동 골목시장  부산광역시  해운대구  부산 해운대구 재반로270번길 43-53       쇼핑   
492  493          광장아울렛  부산광역시   기장군         부산 기장군 정관7로 3-0       쇼핑   
493  494        보수동책방골목  부산광역시    중구                   부산 중구       쇼핑   
496  496         광안종합시장  부산광역시   수영구     부산 수영구 무학로49번길 71-0       쇼핑   

    소분류 카테고리     검색건수  
0        백화점  559,499  
2        백화점  381,864  
6       대형마트  260,036  
9        백화점  172,097  
10       쇼핑몰  152,655  
..       ...      ...  
470     대형마트    2,240  
475       시장    2,179  
492     대형마트    2,059  
493   기타쇼핑시설    2,041  
496       시장    2,033  

[146 rows x 8 columns]

In [12]:
df_nt = df[df['중분류 카테고리'] == '자연관광']
df_nt

순위     관광지명  광역시/도 시/군/구                 도로명주소 중분류 카테고리     소분류 카테고리  \
4      5  광안리해수욕장  부산광역시   수영구    부산 수영구 광안해변로 219-0     자연관광  자연경관(하천/해양)   
5      6  해운대해수욕장  부산광역시  해운대구  부산 해운대구 해운대해변로 264-0     자연관광  자연경관(하천/해양)   
7      8   송정해수욕장  부산광역시  해운대구    부산 해운대구 송정해변로 62-0     자연관광  자연경관(하천/해양)   
12    13  다대포해수욕장  부산광역시   사하구     부산 사하구 몰운대1길 14-0     자연관광  자연경관(하천/해양)   
17    18   송도해수욕장  부산광역시    서구                 부산 서구     자연관광  자연경관(하천/해양)   
..   ...      ...    ...   ...                   ...      ...          ...   
457  458     동명부두  부산광역시    남구        부산 남구 북항로 17-0     자연관광  자연경관(하천/해양)   
467  468     다대포항  부산광역시   사하구                부산 사하구     자연관광  자연경관(하천/해양)   
471  472    신감만부두  부산광역시    남구                 부산 남구     자연관광  자연경관(하천/해양)   
474  475    대항선착장  부산광역시   강서구      부산 강서구 가덕해안로 0-0     자연관광  자연경관(하천/해양)   
480  481     대티고개  부산광역시   사하구                부산 사하구     자연관광      자연경관(산)   

        검색건수  
4    261,818  
5    261,722  
7    244,851  
12   144,834  
17   119,540  
..       ...  
457    2,355  
467    2,257  
471    2,232  
474    2,216  
480    2,158  

[62 rows x 8 columns]

In [13]:
df_etc = df[df['중분류 카테고리'] == '기타관광']
df_etc

순위                  관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리  \
1      2             김해공항국내선청사  부산광역시   강서구      부산 강서구 공항진입로 108-0     기타관광   
3      4                김해국제공항  부산광역시   강서구      부산 강서구 공항진입로 108-0     기타관광   
36    37        해운대블루라인파크미포정거장  부산광역시  해운대구   부산 해운대구 달맞이길62번길 13-0     기타관광   
47    48                 기장해녀촌  부산광역시   기장군       부산 기장군 연화1길 184-0     기타관광   
52    53              송도해상케이블카  부산광역시    서구       부산 서구 송도해변로 171-0     기타관광   
57    58               흰여울문화마을  부산광역시   영도구                  부산 영도구     기타관광   
58    59             부산서부버스터미널  부산광역시   사상구        부산 사상구 사상로 201-0     기타관광   
64    65               해운대달맞이길  부산광역시  해운대구        부산 해운대구 달맞이길 0-0     기타관광   
71    72             부산종합버스터미널  부산광역시   금정구      부산 금정구 중앙대로 2238-0     기타관광   
74    75                   부산역  부산광역시    동구        부산 동구 중앙대로 206-0     기타관광   
81    82             부산신항국제터미널  부산광역시   강서구       부산 강서구 신항남로 330-0     기타관광   
83    84            부산항국제여객터미널  부산광역시    동구        부산 동구 충장대로 206-0     기타관광   
95    96                감천문화마을  부산광역시   사하구       부산 사하구 감내2로 203-0     기타관광   
110  111                   구포역  부산광역시    북구        부산 북구 구포만세길 82-0     기타관광   
118  119                황령산봉수대  부산광역시  부산진구                 부산 부산진구     기타관광   
132  133        해운대블루라인파크송정정거장  부산광역시  해운대구   부산 해운대구 송정중앙로8번길 60-0     기타관광   
156  157       해운대블루라인파크청사포정거장  부산광역시  해운대구      부산 해운대구 청사포로 116-0     기타관광   
220  221               태종대자갈마당  부산광역시   영도구                  부산 영도구     기타관광   
249  250            부산항연안여객터미널  부산광역시    중구         부산 중구 충장대로 24-0     기타관광   
257  258              송도용궁구름다리  부산광역시    서구                   부산 서구     기타관광   
268  269             낙동강하구에코센터  부산광역시   사하구      부산 사하구 낙동남로 1240-0     기타관광   
272  273             해운대유람선터미널  부산광역시  해운대구   부산 해운대구 달맞이길62번길 33-1     기타관광   
276  277                   부전역  부산광역시  부산진구       부산 부산진구 부전로 181-0     기타관광   
289  290             다대포꿈의낙조분수  부산광역시   사하구       부산 사하구 몰운대1길 14-0     기타관광   
307  308  낙동강1경(을숙도생태공원)을숙도문화관  부산광역시   사하구  부산 사하구 낙동남로1233번길 25-0     기타관광   
327  328                이기대섭자리  부산광역시    남구                   부산 남구     기타관광   
331  332                  감지해변  부산광역시   영도구                  부산 영도구     기타관광   
339  340                가덕도선착장  부산광역시   강서구                  부산 강서구     기타관광   
340  341                  부산진역  부산광역시    동구        부산 동구 충장대로 255-0     기타관광   
350  351            해운대시외버스터미널  부산광역시  해운대구      부산 해운대구 해운대로 641-0     기타관광   
362  363              부산항감만터미널  부산광역시    남구         부산 남구 북항로 105-0     기타관광   
364  365                 하리선착장  부산광역시   영도구                  부산 영도구     기타관광   
382  383             부산항신선대터미널  부산광역시    남구         부산 남구 신선로 294-0     기타관광   
404  405               송도스카이워크  부산광역시    서구         부산 서구 송도해변로 0-0     기타관광   
412  413                   사상역  부산광역시   사상구        부산 사상구 광장로 108-0     기타관광   
417  418               흰여울해안터널  부산광역시   영도구                  부산 영도구     기타관광   
447  448             동래시외버스정류소  부산광역시   동래구  부산 동래구 중앙대로1325번길 20-0     기타관광   
455  456                   좌천역  부산광역시   기장군                  부산 기장군     기타관광   
466  467                  대저수문  부산광역시   강서구       부산 강서구 공항로 1516-0     기타관광   
490  491            해운대시외버스정류소  부산광역시  해운대구      부산 해운대구 해운대로 641-0     기타관광   

    소분류 카테고리     검색건수  
1       교통시설  446,879  
3       교통시설  321,903  
36      기타관광   78,421  
47      기타관광   64,940  
52      기타관광   58,215  
57      기타관광   55,152  
58      교통시설   54,181  
64     데이트코스   50,855  
71      교통시설   47,462  
74      교통시설   45,138  
81      교통시설   42,255  
83      교통시설   40,396  
95      기타관광   36,322  
110     교통시설   28,158  
118     기타관광   25,846  
132     기타관광   20,883  
156     기타관광   17,231  
220     기타관광   10,384  
249     교통시설    7,726  
257     기타관광    7,462  
268     기타관광    6,524  
272     교통시설    6,286  
276     교통시설    6,272  
289     기타관광    5,916  
307     기타관광    5,215  
327     기타관광    4,546  
331     기타관광    4,502  
339     교통시설    4,389  
340     교통시설    4,375  
350  

In [15]:
df_sport = df[df['중분류 카테고리'] == '레저스포츠']
df_sport

순위               관광지명  광역시/도 시/군/구                    도로명주소 중분류 카테고리  \
55    56            부산요트경기장  부산광역시  해운대구      부산 해운대구 해운대해변로 84-0    레저스포츠   
56    57          렛츠런파크부산경남  부산광역시   강서구        부산 강서구 가락대로 929-0    레저스포츠   
66    67              해운대CC  부산광역시   기장군        부산 기장군 병산2로 265-0    레저스포츠   
84    85        해운대비치골프앤리조트  부산광역시   기장군          부산 기장군 대변로 74-0    레저스포츠   
91    92            베이사이드GC  부산광역시   기장군        부산 기장군 이천8길 100-0    레저스포츠   
94    95            스톤게이트CC  부산광역시   기장군         부산 기장군 곡천길 317-0    레저스포츠   
96    97          아시아드컨트리클럽  부산광역시   기장군          부산 기장군 차양길 26-0    레저스포츠   
116  117               부산CC  부산광역시   금정구  부산 금정구 중앙대로2327번길 112-0    레저스포츠   
133  134           동래베네스트GC  부산광역시   금정구                   부산 금정구    레저스포츠   
141  142        기장동원로얄컨트리클럽  부산광역시   기장군        부산 기장군 반송로 1345-0    레저스포츠   
169  170         부산북구문화빙상센터  부산광역시    북구      부산 북구 금곡대로46번길 50-0    레저스포츠   
187  188             하이스트CC  부산광역시   강서구      부산 강서구 과학산단1로 172-0    레저스포츠   
199  200               해라CC  부산광역시   강서구   부산 강서구 과학산단로306번길 77-0    레저스포츠   
208  209               문탠로드  부산광역시  해운대구                  부산 해운대구    레저스포츠   
258  259            남구실내빙상장  부산광역시    남구         부산 남구 백운포로 106-0    레저스포츠   
269  270            동래아이스링크  부산광역시   동래구        부산 동래구 충렬대로 420-0    레저스포츠   
274  274           양정청소년수련관  부산광역시  부산진구    부산 부산진구 동평로405번길 85-0    레저스포츠   
280  281          금련산청소년수련원  부산광역시   수영구        부산 수영구 황령산로 156-0    레저스포츠   
294  295            김해체력단련장  부산광역시   강서구                   부산 강서구    레저스포츠   
301  302               서프홀릭  부산광역시  해운대구       부산 해운대구 송정해변로 50-0    레저스포츠   
312  313         BnBK파3골프클럽  부산광역시   금정구                   부산 금정구    레저스포츠   
313  314          스포원파크싸이드롬  부산광역시   금정구  부산 금정구 체육공원로399번길 324-0    레저스포츠   
373  374           회동수원지둘레길  부산광역시   금정구         부산 금정구 개좌로 147-0    레저스포츠   
384  385          기장군청소년수련관  부산광역시   기장군        부산 기장군 달음산길 106-0    레저스포츠   
402  403                더요트  부산광역시  해운대구      부산 해운대구 해운대해변로 84-0    레저스포츠   
407  408           이기대해안산책로  부산광역시    남구                    부산 남구    레저스포츠   
430  431         부산광역시학생교육원  부산광역시   금정구         부산 금정구 북문로 178-0    레저스포츠   
451  452        대저생태공원파크골프장  부산광역시   강서구                   부산 강서구    레저스포츠   
483  484          남해해양경찰수련원  부산광역시   영도구          부산 영도구 와치로 34-0    레저스포츠   
485  486  삼락생태공원인라인스케이트장1트랙  부산광역시   사상구                   부산 사상구    레저스포츠   
498  499            구포무장애숲길  부산광역시    북구                    부산 북구    레저스포츠   

    소분류 카테고리    검색건수  
55   수상레저스포츠  56,185  
56   육상레저스포츠  55,969  
66   육상레저스포츠  50,246  
84   육상레저스포츠  40,325  
91   육상레저스포츠  37,081  
94   육상레저스포츠  36,612  
96   육상레저스포츠  35,804  
116  육상레저스포츠  26,867  
133  육상레저스포츠  20,797  
141  육상레저스포츠  19,434  
169  육상레저스포츠  15,833  
187  육상레저스포츠  12,849  
199  육상레저스포츠  11,849  
208  기타레저스포츠  11,144  
258  육상레저스포츠   7,447  
269  육상레저스포츠   6,444  
274  육상레저스포츠   6,279  
280  육상레저스포츠   6,162  
294  육상레저스포츠   5,752  
301  수상레저스포츠   5,367  
312  육상레저스포츠   5,038  
313  육상레저스포츠   5,014  
373  기타레저스포츠   3,620  
384  육상레저스포츠   3,399  
402  기타레저스포츠   3,051  
407  기타레저스포츠   2,963  
430  육상레저스포츠   2,625  
451  육상레저스포츠   2,403  
483  육상레저스포츠   2,126  
485  육상레저스포츠   2,094  
498  기타레저스포츠   2,025

In [16]:
df_ht = df[df['중분류 카테고리'] == '역사관광']
df_ht

순위    관광지명  광역시/도 시/군/구                    도로명주소 중분류 카테고리 소분류 카테고리  \
8      9   해동용궁사  부산광역시   기장군          부산 기장군 용궁길 86-0     역사관광     종교성지   
16    17     태종대  부산광역시   영도구          부산 영도구 전망로 24-0     역사관광    역사유적지   
41    42     범어사  부산광역시   금정구        부산 금정구 범어사로 250-0     역사관광     종교성지   
78    79     장안사  부산광역시   기장군         부산 기장군 장안로 482-0     역사관광     종교성지   
131  132     삼광사  부산광역시  부산진구    부산 부산진구 초읍천로43번길 77-0     역사관광     종교성지   
186  187     충렬사  부산광역시   동래구        부산 동래구 충렬대로 345-0     역사관광    역사유적지   
191  192     홍법사  부산광역시   금정구     부산 금정구 두구로33번길 202-0     역사관광     종교성지   
193  194     해광사  부산광역시   기장군       부산 기장군 기장해안로 340-0     역사관광     종교성지   
200  201     선암사  부산광역시  부산진구       부산 부산진구 백양산로 138-0     역사관광     종교성지   
265  266    해운정사  부산광역시  해운대구        부산 해운대구 우동2로 40-6     역사관광     종교성지   
296  297     소원사  부산광역시   기장군       부산 기장군 정관중앙로 198-0     역사관광     종교성지   
318  319     운수사  부산광역시   사상구    부산 사상구 모라로219번길 173-0     역사관광     종교성지   
341  342     정수사  부산광역시   연제구         부산 연제구 톳고개로 60-0     역사관광     종교성지   
342  342    혜원정사  부산광역시   연제구      부산 연제구 고분로68번길 47-0     역사관광     종교성지   
344  345    동명불원  부산광역시    남구           부산 남구 동명로 57-0     역사관광     종교성지   
351  352     은진사  부산광역시   기장군        부산 기장군 고무로 271-47     역사관광     종교성지   
352  353    내원정사  부산광역시    서구      부산 서구 엄광산로40번길 80-0     역사관광     종교성지   
359  360     옥정사  부산광역시   기장군         부산 기장군 달음길 101-0     역사관광     종교성지   
365  366    금정산성  부산광역시   금정구                   부산 금정구     역사관광    역사유적지   
421  422     해월정  부산광역시  해운대구                  부산 해운대구     역사관광    역사유적지   
432  433     마하사  부산광역시   연제구         부산 연제구 봉수로 138-0     역사관광     종교성지   
436  437  금정산성동문  부산광역시   금정구                   부산 금정구     역사관광    역사유적지   
446  447     광명사  부산광역시   금정구     부산 금정구 금샘로17번안길 45-0     역사관광     종교성지   
448  449     성불사  부산광역시  해운대구  부산 해운대구 해운대로383번길 270-0     역사관광     종교성지   
458  459   해동성취사  부산광역시   기장군         부산 기장군 일광로 427-0     역사관광     종교성지   
477  476     인각사  부산광역시    남구          부산 남구 진남로 234-0     역사관광     종교성지   
479  480    안국선원  부산광역시   금정구         부산 금정구 금단로 124-0     역사관광     종교성지   
484  485    옥련선원  부산광역시   수영구     부산 수영구 광남로257번길 58-0     역사관광     종교성지   
486  487     복천사  부산광역시   영도구          부산 영도구 산정길 41-0     역사관광     종교성지   

        검색건수  
8    199,253  
16   121,311  
41    73,779  
78    43,401  
131   21,066  
186   13,143  
191   12,398  
193   12,275  
200   11,794  
265    6,921  
296    5,699  
318    4,862  
341    4,374  
342    4,374  
344    4,309  
351    4,187  
352    4,132  
359    3,979  
365    3,844  
421    2,758  
432    2,606  
436    2,581  
446    2,426  
448    2,415  
458    2,351  
477    2,179  
479    2,159  
484    2,120  
486    2,092

In [20]:
df2

순위            관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리  \
0      1            칠암사계  부산광역시   기장군        부산 기장군 칠암1길 7-10       음식   
1      2          웨이브온커피  부산광역시   기장군       부산 기장군 해맞이로 286-0       음식   
2      3      피아크카페앤베이커리  부산광역시   영도구   부산 영도구 해양로195번길 180-0       음식   
3      4        해운대암소갈비집  부산광역시  해운대구  부산 해운대구 중동2로10번길 32-10       음식   
4      5        합천일류돼지국밥  부산광역시   사상구         부산 사상구 광장로 34-0       음식   
..   ...             ...    ...   ...                     ...      ...   
495  496             하동집  부산광역시   금정구       부산 금정구 오륜대로 258-0       음식   
496  497  맛찬들왕소금구이대연동직영점  부산광역시    남구        부산 남구 유엔평화로 36-0       음식   
497  498      탐앤탐스블랙달맞이점  부산광역시  해운대구      부산 해운대구 달맞이길 213-0       음식   
498  499             버거샵  부산광역시  해운대구   부산 해운대구 우동1로20번길 19-0       음식   
499  500         바다마루전복죽  부산광역시  해운대구    부산 해운대구 달맞이길62번길 7-0       음식   

    소분류 카테고리     검색건수  
0      카페/찻집  100,578  
1      카페/찻집   66,691  
2      카페/찻집   60,904  
3         한식   50,002  
4         한식   35,188  
..       ...      ...  
495     전문음식    3,774  
496       한식    3,762  
497    카페/찻집    3,761  
498     간이음식    3,749  
499     전문음식    3,732  

[500 rows x 8 columns]

In [21]:
df3

순위       관광지명  광역시/도 시/군/구                     도로명주소 중분류 카테고리 소분류 카테고리  \
0      1    아난티힐튼부산  부산광역시   기장군       부산 기장군 기장해안로 268-32       숙박       호텔   
1      2  파라다이스호텔부산  부산광역시  해운대구      부산 해운대구 해운대해변로 296-0       숙박       호텔   
2      3   한화리조트해운대  부산광역시  해운대구       부산 해운대구 마린시티3로 52-0       숙박    콘도미니엄   
3      4  웨스틴조선호텔부산  부산광역시  해운대구          부산 해운대구 동백로 67-0       숙박       호텔   
4      5    롯데호텔부산점  부산광역시  부산진구        부산 부산진구 가야대로 772-0       숙박       호텔   
..   ...        ...    ...   ...                       ...      ...      ...   
495  496   하운드호텔정관점  부산광역시   기장군         부산 기장군 산단1로 98-11       숙박       호텔   
496  497       모모쥬스  부산광역시  해운대구  부산 해운대구 해운대해변로209번길 12-0       숙박       모텔   
497  497      아이유모텔  부산광역시   연제구     부산 연제구 월드컵대로99번길 31-0       숙박       모텔   
498  499       제이모텔  부산광역시   사상구          부산 사상구 대동로 297-0       숙박       모텔   
499  500       브이호텔  부산광역시    중구            부산 중구 보수대로 8-3       숙박       호텔   

        검색건수  
0    152,941  
1    111,872  
2    103,401  
3     62,558  
4     60,070  
..       ...  
495      983  
496      978  
497      978  
498      976  
499      971  

[500 rows x 8 columns]

In [22]:
df4

순번          가맹점명  \
0            1        고봉민김밥인   
1            2      미사랑스킨앤바디   
2            3    피자스쿨 부산재송점   
3            4  미스터피자아시아드2호점   
4            5         성원칼국수   
...        ...           ...   
126936  126937    행복한 정쌤 공부방   
126937  126938         행복홈케어   
126938  126939         현대표구사   
126939  126940          현혜반점   
126940  126941   개인택시조합 구내식당   

                                                     도로명                  지번  \
0                                부산 영도구 와치로 250 (동삼동) 1층                 NaN   
1                            부산 사하구 승학로224번길 10 ,1층(괴정동)                 NaN   
2                    부산 해운대구 재반로 182 비둘기맨션상가 B동 가호 (재송동)                 NaN   
3                         부산 연제구 월드컵대로 232 2층 (거제동,거벌빌등)                 NaN   
4                                부산 부산진구 냉정로 220-1 (개금동)                 NaN   
...                                                  ...                 ...   
126936  부산 강서구  명지오션시티11로 22, 102동 203호 (명지동,명지두산위브포세이돈)                 NaN   
126937        부산 사하구 서포로30번길 12 207동 301호 (구평동,e편한세상아파트)                 NaN   
126938                         부산 중구  중앙대로 137-1 (대창동2가)                 NaN   
126939                                               NaN  부산 사상구 덕포동 404-7,8   
126940                            부산 연제구 월드컵대로 399 (거제동)                 NaN   

               위도          경도     데이터기준일자  
0       35.075315  129.068770  2022-12-22  
1       35.101239  128.990313  2022-12-22  
2       35.193467  129.130557  2022-12-22  
3       35.189890  129.071940  2022-12-22  
4       35.150897  129.023928  2022-12-22  
...           ...         ...         ...  
126936  35.083390  128.896842  2022-12-22  
126937  35.081755  128.985395  2022-12-22  
126938  35.109934  129.037028  2022-12-22  
126939        NaN         NaN  2022-12-22  
126940  35.186010  129.055979  2022-12-22  

[126941 rows x 7 columns]

In [56]:
df_et = df[df['중분류 카테고리'] == '체험관광']
df_et

순위      관광지명  광역시/도 시/군/구                  도로명주소 중분류 카테고리  소분류 카테고리  \
53    54       허심청  부산광역시   동래구  부산 동래구 온천장로107번길 32-0     체험관광     웰니스관광   
205  206  센텀시티스파랜드  부산광역시  해운대구     부산 해운대구 센텀남대로 35-0     체험관광     웰니스관광   
237  238   해운대온천센터  부산광역시  해운대구      부산 해운대구 중동2로 16-0     체험관광     웰니스관광   
262  263      정관온천  부산광역시   기장군       부산 기장군 정관8로 16-5     체험관광     웰니스관광   
325  326     태종대온천  부산광역시   영도구       부산 영도구 태종로 808-0     체험관광     웰니스관광   
403  404      카덴96  부산광역시   기장군     부산 기장군 기장대로 1075-3     체험관광     웰니스관광   
410  410   금정산부곡온천  부산광역시   금정구    부산 금정구 수림로20번길 36-0     체험관광     웰니스관광   
414  415  공수어촌체험마을  부산광역시   기장군      부산 기장군 공수해안길 13-2     체험관광  농/산/어촌체험   
488  489     라이프온천  부산광역시   기장군      부산 기장군 반송로 1650-0     체험관광     웰니스관광   

       검색건수  
53   57,842  
205  11,326  
237   8,554  
262   7,210  
325   4,664  
403   3,050  
410   2,889  
414   2,868  
488   2,073

In [25]:
df4[df4['가맹점명'].str.contains('허심청')]

순번        가맹점명                        도로명   지번         위도  \
37241  37242  하삼동커피 허심청점  부산 동래구 온천장로 81-1 1층 (온천동)  NaN  35.218633   
68716  68717   메가커피 허심청점      부산 동래구 온천장로 125 (온천동)  NaN  35.221874   

               경도     데이터기준일자  
37241  129.082270  2022-12-22  
68716  129.084784  2022-12-22

In [9]:
df4[df4['가맹점명'].str.contains('해운대해수욕장')]

순번             가맹점명                    도로명   지번         위도  \
44749  44750  다비치안경해운대해수욕장입구점  부산 해운대구 구남로 11-1 (우동)  NaN  35.162876   

               경도     데이터기준일자  
44749  129.159814  2022-12-22

In [58]:
df4[df4['가맹점명'].str.contains('정관온천')]

순번      가맹점명                                      도로명   지번  \
56883  56884  정관온천주식회사  부산 기장군 정관면 정관 8 로 16-5, ( 매학리 , 정관온천 )   NaN   

              위도         경도     데이터기준일자  
56883  35.332765  129.17678  2022-12-22

# ㅁㄴㅇㄹ

In [5]:
result1 = pd.concat([df, df2])
result2 = pd.concat([result1, df3])
result2

순위         관광지명  광역시/도 시/군/구                     도로명주소 중분류 카테고리  \
0      1  신세계백화점센텀시티점  부산광역시  해운대구        부산 해운대구 센텀남대로 35-0       쇼핑   
1      2    김해공항국내선청사  부산광역시   강서구        부산 강서구 공항진입로 108-0     기타관광   
2      3    롯데백화점부산본점  부산광역시  부산진구        부산 부산진구 가야대로 772-0       쇼핑   
3      4       김해국제공항  부산광역시   강서구        부산 강서구 공항진입로 108-0     기타관광   
4      5      광안리해수욕장  부산광역시   수영구        부산 수영구 광안해변로 219-0     자연관광   
..   ...          ...    ...   ...                       ...      ...   
495  496     하운드호텔정관점  부산광역시   기장군         부산 기장군 산단1로 98-11       숙박   
496  497         모모쥬스  부산광역시  해운대구  부산 해운대구 해운대해변로209번길 12-0       숙박   
497  497        아이유모텔  부산광역시   연제구     부산 연제구 월드컵대로99번길 31-0       숙박   
498  499         제이모텔  부산광역시   사상구          부산 사상구 대동로 297-0       숙박   
499  500         브이호텔  부산광역시    중구            부산 중구 보수대로 8-3       숙박   

        소분류 카테고리     검색건수  
0            백화점  559,499  
1           교통시설  446,879  
2            백화점  381,864  
3           교통시설  321,903  
4    자연경관(하천/해양)  261,818  
..           ...      ...  
495           호텔      983  
496           모텔      978  
497           모텔      978  
498           모텔      976  
499           호텔      971  

[1500 rows x 8 columns]

In [6]:
result2['검색건수'] = result2['검색건수'].str.replace(',','')
result2['검색건수'] = result2['검색건수'].astype('int')

In [7]:
df_new = result2.sort_values(by='검색건수',ascending = False)
df_new.head(50)

순위            관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리  \
0    1     신세계백화점센텀시티점  부산광역시  해운대구      부산 해운대구 센텀남대로 35-0       쇼핑   
1    2       김해공항국내선청사  부산광역시   강서구      부산 강서구 공항진입로 108-0     기타관광   
2    3       롯데백화점부산본점  부산광역시  부산진구      부산 부산진구 가야대로 772-0       쇼핑   
3    4          김해국제공항  부산광역시   강서구      부산 강서구 공항진입로 108-0     기타관광   
4    5         광안리해수욕장  부산광역시   수영구      부산 수영구 광안해변로 219-0     자연관광   
5    6         해운대해수욕장  부산광역시  해운대구    부산 해운대구 해운대해변로 264-0     자연관광   
6    7      코스트코홀세일부산점  부산광역시   수영구        부산 수영구 구락로 137-0       쇼핑   
7    8          송정해수욕장  부산광역시  해운대구      부산 해운대구 송정해변로 62-0     자연관광   
8    9           해동용궁사  부산광역시   기장군         부산 기장군 용궁길 86-0     역사관광   
9   10        롯데백화점광복점  부산광역시    중구          부산 중구 중앙대로 2-0       쇼핑   
0    1         아난티힐튼부산  부산광역시   기장군     부산 기장군 기장해안로 268-32       숙박   
10  11   롯데프리미엄아울렛동부산점  부산광역시   기장군      부산 기장군 기장해안로 147-0       쇼핑   
11  12        NC백화점서면점  부산광역시  부산진구        부산 부산진구 동천로 92-0       쇼핑   
12  13         다대포해수욕장  부산광역시   사하구       부산 사하구 몰운대1길 14-0     자연관광   
13  14      롯데월드어드벤처부산  부산광역시   기장군      부산 기장군 동부산관광로 42-0     문화관광   
14  15     이마트트레이더스연산점  부산광역시   연제구       부산 연제구 좌수영로 241-0       쇼핑   
15  16            기장시장  부산광역시   기장군    부산 기장군 읍내로104번길 16-0       쇼핑   
16  17             태종대  부산광역시   영도구         부산 영도구 전망로 24-0     역사관광   
17  18          송도해수욕장  부산광역시    서구                   부산 서구     자연관광   
18  19        롯데백화점동래점  부산광역시   동래구      부산 동래구 중앙대로 1393-0       쇼핑   
19  20       반여농산물도매시장  부산광역시  해운대구    부산 해운대구 수영강변대로 626-0       쇼핑   
20  21            국제시장  부산광역시    중구       부산 중구 국제시장2길 25-0       쇼핑   
1    2       파라다이스호텔부산  부산광역시  해운대구    부산 해운대구 해운대해변로 296-0       숙박   
21  22         이케아동부산점  부산광역시   기장군     부산 기장군 동부산관광3로 17-0       쇼핑   
2    3        한화리조트해운대  부산광역시  해운대구     부산 해운대구 마린시티3로 52-0       숙박   
0    1            칠암사계  부산광역시   기장군        부산 기장군 칠암1길 7-10       음식   
22  23     이마트트레이더스명지점  부산광역시   강서구     부산 강서구 명지국제6로 168-0       쇼핑   
23  24           자갈치시장  부산광역시    중구       부산 중구 자갈치해안로 52-0       쇼핑   
24  25         이마트해운대점  부산광역시  해운대구     부산 해운대구 좌동순환로 511-0       쇼핑   
25  26          이마트문현점  부산광역시    남구     부산 남구 전포대로91번길 47-0       쇼핑   
26  27         롯데마트화명점  부산광역시    북구         부산 북구 화명대로 47-0       쇼핑   
27  28           BEXCO  부산광역시  해운대구      부산 해운대구 APEC로 55-0     문화관광   
28  29   신세계프리미엄아울렛부산점  부산광역시   기장군       부산 기장군 정관로 1133-0       쇼핑   
29  30        신세계센텀시티몰  부산광역시  해운대구       부산 해운대구 센텀4로 15-0       쇼핑   
30  31        벡스코제1전시장  부산광역시  해운대구      부산 해운대구 APEC로 55-0     문화관광   
31  32         메가마트동래점  부산광역시   동래구       부산 동래구 충렬대로 197-0       쇼핑   
32  33     이마트트레이더스서면점  부산광역시  부산진구      부산 부산진구 시민공원로 31-0       쇼핑   
33  34          이마트연제점  부산광역시   연제구         부산 연제구 연수로 89-0       쇼핑   
34  35          일광해수욕장  부산광역시   기장군         부산 기장군 삼성3길 0-0     자연관광   
35  36          부산시민공원  부산광역시  부산진구      부산 부산진구 시민공원로 73-0     문화관광   
36  37  해운대블루라인파크미포정거장  부산광역시  해운대구   부산 해운대구 달맞이길62번길 13-0     기타관광   
37  38          이마트금정점  부산광역시   금정구  부산 금정구 중앙대로1841번길 24-0       쇼핑   
38  39      롯데백화점센텀시티점  부산광역시  해운대구      부산 해운대구 센텀남대로 59-0       쇼핑   
39  40       홈플러스아시아드점  부산광역시   연제구       부산 연제구 종합운동장로 7-0       쇼핑   
40  41             대변항  부산광역시   기장군        부산 기장군 기장해안로 0-0     자연관광   
41  42             범어사  부산광역시   금정구       부산 금정구 범어사로 250-0     역사관광   
42  43          이마트사상점  부산광역시   사상구         부산 사상구 광장로 17-0       쇼핑   
43  44           신선대부두  부산광역시    남구                   부산 남구     자연관광   
44  45        현대백화점부산점  부산광역시    동구         부산 동구 범일로 125-0       쇼핑   
45  46       엄궁농산물도매시장  부산광역시   사상구   부산 사상구 농산물시장로9번길 11-0       쇼핑   

       소분류 카테고리    검색건수  
0           백화점  559499  
1          교통시설  446879  
2           백화점  381864  
3          교통시설  321903  
4   자연경관(하천/해양)  261818  
5   자연경관(하천/해양)  261722  
6          대형마트  260036  
7   자연경관(하천/해양)  244851  
8          종교성지  199253  
9           백화점  172097  
0            호텔  152941  
10          쇼핑몰  15

In [10]:
df_new.describe()

순위           검색건수
count  1500.000000    1500.000000
mean    250.460000   13113.522000
std     144.358281   32055.719468
min       1.000000     971.000000
25%     125.750000    2745.750000
50%     250.500000    5045.500000
75%     375.250000   10157.000000
max     500.000000  559499.000000

# api

In [13]:
import requests

url = 'http://apis.data.go.kr/6260000/BusanTourStaticService2/getVisitorStatInfo2'
params ={'serviceKey' : '0GFThSrrIYeZWEGew0ybzsOdq6pANbVrZE9nAcjjrqO4hpPyF4/HS5MaRgb83oizsBHn7hY5ek8MYcL52KwIUA==', 'numOfRows' : '10', 'pageNo' : '1', 'resultType' : 'xml' }

response = requests.get(url, params=params).text
response

'<?xml version="1.0" encoding="UTF-8"?><response>\n  <header>\n    <resultMsg>NORMAL_CODE</resultMsg>\n    <resultCode>00</resultCode>\n  </header>\n  <body>\n    <items>\n      <item>\n        <car>118393204</car>\n        <grocery>460363898</grocery>\n        <provincejeju>37449</provincejeju>\n        <provincedaegu>157822</provincedaegu>\n        <provincejeonbuk>33842</provincejeonbuk>\n        <clothing>2586385764</clothing>\n        <sports>1879219481</sports>\n        <provincegyeongbuk>102721</provincegyeongbuk>\n        <provinceincheon>93496</provinceincheon>\n        <travel>149356280</travel>\n        <beauty>921787813</beauty>\n        <online>25677805</online>\n        <spot>BIFF광장·용두산공원,보수동책방골목</spot>\n        <agethirties>383953</agethirties>\n        <provincegyeongnam>345244</provincegyeongnam>\n        <medical>958692434</medical>\n        <provincechungbuk>32664</provincechungbuk>\n        <agesixties>299052</agesixties>\n        <agetwenties>400169</agetwenties>\n

# 123

In [ ]:
df2.head()

순번          가맹점명                                  도로명   지번         위도  \
0   1        고봉민김밥인              부산 영도구 와치로 250 (동삼동) 1층  NaN  35.075315   
1   2      미사랑스킨앤바디          부산 사하구 승학로224번길 10 ,1층(괴정동)  NaN  35.101239   
2   3    피자스쿨 부산재송점  부산 해운대구 재반로 182 비둘기맨션상가 B동 가호 (재송동)  NaN  35.193467   
3   4  미스터피자아시아드2호점       부산 연제구 월드컵대로 232 2층 (거제동,거벌빌등)  NaN  35.189890   
4   5         성원칼국수              부산 부산진구 냉정로 220-1 (개금동)  NaN  35.150897   

           경도     데이터기준일자  
0  129.068770  2022-12-22  
1  128.990313  2022-12-22  
2  129.130557  2022-12-22  
3  129.071940  2022-12-22  
4  129.023928  2022-12-22

In [ ]:
df_cul_nat = df[(df['중분류 카테고리'] == '문화관광') | (df['중분류 카테고리'] == '자연관광')]
df_cul_nat

순위        관광지명  광역시/도 시/군/구                 도로명주소 중분류 카테고리     소분류 카테고리  \
4      5     광안리해수욕장  부산광역시   수영구    부산 수영구 광안해변로 219-0     자연관광  자연경관(하천/해양)   
5      6     해운대해수욕장  부산광역시  해운대구  부산 해운대구 해운대해변로 264-0     자연관광  자연경관(하천/해양)   
7      8      송정해수욕장  부산광역시  해운대구    부산 해운대구 송정해변로 62-0     자연관광  자연경관(하천/해양)   
12    13     다대포해수욕장  부산광역시   사하구     부산 사하구 몰운대1길 14-0     자연관광  자연경관(하천/해양)   
13    14  롯데월드어드벤처부산  부산광역시   기장군    부산 기장군 동부산관광로 42-0     문화관광         테마공원   
..   ...         ...    ...   ...                   ...      ...          ...   
491  492        75광장  부산광역시   영도구                부산 영도구     문화관광         도시공원   
494  495      조은극장1관  부산광역시    중구     부산 중구 구덕로34번길 4-0     문화관광         공연시설   
495  496     메가박스해운대  부산광역시  해운대구    부산 해운대구 해운대로 813-0     문화관광         공연시설   
497  498     메가박스부산대  부산광역시   금정구   부산 금정구 장전로12번길 55-0     문화관광         공연시설   
499  500      비콘그라운드  부산광역시   수영구      부산 수영구 망미번영로 0-0     문화관광      기타문화관광지   

        검색건수  
4    261,818  
5    261,722  
7    244,851  
12   144,834  
13   134,067  
..       ...  
491    2,060  
494    2,040  
495    2,033  
497    2,029  
499    2,015  

[245 rows x 8 columns]

In [ ]:
df2[df2['관광지명'].str.contains('광안리')]

순위       관광지명  광역시/도 시/군/구                  도로명주소 중분류 카테고리 소분류 카테고리  \
213  214  커피스미스광안리점  부산광역시   수영구  부산 수영구 광안해변로255번길 5-0       음식    카페/찻집   
397  398   스타벅스광안리점  부산광역시   수영구     부산 수영구 광안해변로 247-0       음식    카페/찻집   

      검색건수  
213  7,139  
397  4,540

In [ ]:
df2['소분류 카테고리'].value_counts()

한식       162
카페/찻집    131
전문음식     110
간이음식      59
외국식       37
음식점기타      1
Name: 소분류 카테고리, dtype: int64

In [ ]:
df_cafe = df2[df2['소분류 카테고리'] == '카페/찻집']
df_cafe

순위         관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리 소분류 카테고리  \
0      1         칠암사계  부산광역시   기장군        부산 기장군 칠암1길 7-10       음식    카페/찻집   
1      2       웨이브온커피  부산광역시   기장군       부산 기장군 해맞이로 286-0       음식    카페/찻집   
2      3   피아크카페앤베이커리  부산광역시   영도구   부산 영도구 해양로195번길 180-0       음식    카페/찻집   
7      8  스타벅스명지강변DT점  부산광역시   강서구     부산 강서구 르노삼성대로 620-0       음식    카페/찻집   
10    11          헤이든  부산광역시   기장군        부산 기장군 문오성길 22-0       음식    카페/찻집   
..   ...          ...    ...   ...                     ...      ...      ...   
479  480          쿼피집  부산광역시   사상구    부산 사상구 사상로356번길 16-0       음식    카페/찻집   
480  481         로와맨션  부산광역시   기장군         부산 기장군 학리2길 6-0       음식    카페/찻집   
487  488          아시오  부산광역시  부산진구  부산 부산진구 월드컵대로472번길 1-0       음식    카페/찻집   
493  494   카페노티드부산해운대  부산광역시  해운대구        부산 해운대구 구남로 17-0       음식    카페/찻집   
497  498   탐앤탐스블랙달맞이점  부산광역시  해운대구      부산 해운대구 달맞이길 213-0       음식    카페/찻집   

        검색건수  
0    100,578  
1     66,691  
2     60,904  
7     32,333  
10    30,271  
..       ...  
479    3,874  
480    3,871  
487    3,821  
493    3,782  
497    3,761  

[131 rows x 8 columns]

In [ ]:
df_cafe['지역화폐 사용가능'] = 'o'

<ipython-input-11-b93d5880b6d9>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_cafe['지역화폐 사용가능'] = 'o'


In [ ]:
df_cafe

순위         관광지명  광역시/도 시/군/구                   도로명주소 중분류 카테고리 소분류 카테고리  \
0      1         칠암사계  부산광역시   기장군        부산 기장군 칠암1길 7-10       음식    카페/찻집   
1      2       웨이브온커피  부산광역시   기장군       부산 기장군 해맞이로 286-0       음식    카페/찻집   
2      3   피아크카페앤베이커리  부산광역시   영도구   부산 영도구 해양로195번길 180-0       음식    카페/찻집   
7      8  스타벅스명지강변DT점  부산광역시   강서구     부산 강서구 르노삼성대로 620-0       음식    카페/찻집   
10    11          헤이든  부산광역시   기장군        부산 기장군 문오성길 22-0       음식    카페/찻집   
..   ...          ...    ...   ...                     ...      ...      ...   
479  480          쿼피집  부산광역시   사상구    부산 사상구 사상로356번길 16-0       음식    카페/찻집   
480  481         로와맨션  부산광역시   기장군         부산 기장군 학리2길 6-0       음식    카페/찻집   
487  488          아시오  부산광역시  부산진구  부산 부산진구 월드컵대로472번길 1-0       음식    카페/찻집   
493  494   카페노티드부산해운대  부산광역시  해운대구        부산 해운대구 구남로 17-0       음식    카페/찻집   
497  498   탐앤탐스블랙달맞이점  부산광역시  해운대구      부산 해운대구 달맞이길 213-0       음식    카페/찻집   

        검색건수 지역화폐 사용가능  
0    100,578         o  
1     66,691         o  
2     60,904         o  
7     32,333         o  
10    30,271         o  
..       ...       ...  
479    3,874         o  
480    3,871         o  
487    3,821         o  
493    3,782         o  
497    3,761         o  

[131 rows x 9 columns]

In [ ]:
df_cafe.head(10)

순위         관광지명  광역시/도 시/군/구                  도로명주소 중분류 카테고리 소분류 카테고리  \
0    1         칠암사계  부산광역시   기장군       부산 기장군 칠암1길 7-10       음식    카페/찻집   
1    2       웨이브온커피  부산광역시   기장군      부산 기장군 해맞이로 286-0       음식    카페/찻집   
2    3   피아크카페앤베이커리  부산광역시   영도구  부산 영도구 해양로195번길 180-0       음식    카페/찻집   
7    8  스타벅스명지강변DT점  부산광역시   강서구    부산 강서구 르노삼성대로 620-0       음식    카페/찻집   
10  11          헤이든  부산광역시   기장군       부산 기장군 문오성길 22-0       음식    카페/찻집   
13  14          비아조  부산광역시   강서구       부산 강서구 식만로 164-0       음식    카페/찻집   
15  16         코랄라니  부산광역시   기장군      부산 기장군 기장해안로 32-0       음식    카페/찻집   
19  20        피크스퀘어  부산광역시   기장군     부산 기장군 기장해안로 864-0       음식    카페/찻집   
22  23   스타벅스동명대DT점  부산광역시    남구        부산 남구 신선로 423-0       음식    카페/찻집   
23  24  모모스로스터리&커피바  부산광역시   영도구     부산 영도구 봉래나루로 160-0       음식    카페/찻집   

       검색건수 지역화폐 사용가능  
0   100,578         o  
1    66,691         o  
2    60,904         o  
7    32,333         o  
10   30,271         o  
13   28,549         o  
15   25,349         o  
19   22,582         o  
22   20,157         o  
23   19,923         o

In [ ]:
df_starbuck = df_cafe[df_cafe['관광지명'].str.contains('스타벅스')]
df_starbuck

순위           관광지명  광역시/도 시/군/구                  도로명주소 중분류 카테고리 소분류 카테고리  \
7      8    스타벅스명지강변DT점  부산광역시   강서구    부산 강서구 르노삼성대로 620-0       음식    카페/찻집   
22    23     스타벅스동명대DT점  부산광역시    남구        부산 남구 신선로 423-0       음식    카페/찻집   
35    36  스타벅스해운대달맞이DT점  부산광역시  해운대구      부산 해운대구 달맞이길 57-0       음식    카페/찻집   
38    39    스타벅스화명강변DT점  부산광역시    북구        부산 북구 금곡대로 77-0       음식    카페/찻집   
46    47     스타벅스동부산DT점  부산광역시   기장군      부산 기장군 기장해안로 56-0       음식    카페/찻집   
56    57    스타벅스부산재송DT점  부산광역시  해운대구     부산 해운대구 해운대로 189-0       음식    카페/찻집   
59    60     스타벅스기장역DT점  부산광역시   기장군      부산 기장군 기장대로 527-0       음식    카페/찻집   
60    61   스타벅스을숙도강변DT점  부산광역시   사하구     부산 사하구 하신번영로 381-6       음식    카페/찻집   
101  102    스타벅스사상주례DT점  부산광역시   사상구      부산 사상구 백양대로 520-0       음식    카페/찻집   
119  120     스타벅스동서대DT점  부산광역시   사상구      부산 사상구 가야대로 338-0       음식    카페/찻집   
133  133    스타벅스수영망미DT점  부산광역시   수영구        부산 수영구 과정로 68-0       음식    카페/찻집   
151  152    스타벅스동래미남DT점  부산광역시   동래구      부산 동래구 충렬대로 101-0       음식    카페/찻집   
159  160    스타벅스부산송정비치점  부산광역시  해운대구     부산 해운대구 송정해변로 36-0       음식    카페/찻집   
161  162    스타벅스부산안락DT점  부산광역시   동래구      부산 동래구 충렬대로 446-0       음식    카페/찻집   
162  163    스타벅스수영강변DT점  부산광역시  해운대구   부산 해운대구 수영강변대로 516-0       음식    카페/찻집   
170  171    스타벅스금정구청DT점  부산광역시   금정구     부산 금정구 중앙대로 1730-0       음식    카페/찻집   
176  177    스타벅스온천장역DT점  부산광역시   금정구       부산 금정구 식물원로 13-0       음식    카페/찻집   
179  180   스타벅스금정남산동DT점  부산광역시   금정구     부산 금정구 중앙대로 1989-0       음식    카페/찻집   
181  182      스타벅스부산정관점  부산광역시   기장군       부산 기장군 정관로 579-0       음식    카페/찻집   
212  212    스타벅스영도청학DT점  부산광역시   영도구       부산 영도구 태종로 373-0       음식    카페/찻집   
223  224    스타벅스삼락강변DT점  부산광역시   사상구     부산 사상구 낙동대로 1306-0       음식    카페/찻집   
241  242    스타벅스해운대달맞이점  부산광역시  해운대구     부산 해운대구 달맞이길 189-0       음식    카페/찻집   
247  248   스타벅스용호동W스퀘어점  부산광역시    남구        부산 남구 분포로 145-0       음식    카페/찻집   
259  260   스타벅스부산교대역DT점  부산광역시   연제구     부산 연제구 중앙대로 1250-0       음식    카페/찻집   
263  264    스타벅스사하장림DT점  부산광역시   사하구     부산 사하구 을숙도대로 682-0       음식    카페/찻집   
302  303      스타벅스영도대교점  부산광역시   영도구        부산 영도구 태종로 39-0       음식    카페/찻집   
311  312    스타벅스명지오션시티점  부산광역시   강서구  부산 강서구 명지오션시티11로 66-0       음식    카페/찻집   
379  380   스타벅스명지국제신도시점  부산광역시   강서구     부산 강서구 명지국제5로 42-0       음식    카페/찻집   
397  398       스타벅스광안리점  부산광역시   수영구     부산 수영구 광안해변로 247-0       음식    카페/찻집   
422  423    스타벅스사상엄궁DT점  부산광역시   사상구      부산 사상구 낙동대로 744-0       음식    카페/찻집   
427  428     스타벅스부산덕천역점  부산광역시    북구        부산 북구 만덕대로 30-0       음식    카페/찻집   
439  440    스타벅스부산만덕DT점  부산광역시    북구       부산 북구 만덕대로 323-0       음식    카페/찻집   

       검색건수 지역화폐 사용가능  
7    32,333         x  
22   20,157         o  
35   17,057         o  
38   16,527         o  
46   14,818         o  
56   13,393         o  
59   13,221         o  
60   13,209         o  
101  10,362         o  
119   9,572         o  
133   9,183         o  
151   8,622         o  
159   8,458         o  
161   8,445         o  
162   8,409         o  
170   8,276         o  
176   8,110         o  
179   8,077         o  
181   8,032         o  
212   7,153         o  
223   6,854         o  
241   6,536         o  
247   6,491         o  
259   6,238         o  
263   6,234         o  
302   5,552         o  
311   5,515         o  
379   4,733         o  
397   4,540         o  
422   4,303         o  
427   4,244         o  
439   4,182         o

카페 : 131개

* 지역화폐 사용 불가능
    * 스타벅스 : 32개
    * 코랄라니

* 애매한 것
    * 헤이든